lib/proportions.js

```diff
@@ -107,16 +109,34 @@ define(['d3-interpolate', 'virtual-dom', 'filters/genericnode', 'helper'],
           return _.t('node.deactivated');
         });
 
+        nodes.forEach(function (node) {
+          if (node.flags.gateway) {
+            var mesh = node.nodeinfo.network.mesh;
+            mesh[Object.keys(mesh)[0]].interfaces.tunnel.forEach(function (mac) {
```

Comment:
I agree a static one is also bad.
Maybe there should a solution if there are more the one mesh interface.
I am not sure if it is better then they are three loops nested.

Possible this is already the best way.